cli/src/ansible/AnsibleVarsGenerator.py

```diff
@@ -120,11 +120,11 @@ def write_role_vars(self, ansible_dir, role, document, vars_file_name='main.yml'
 
     def write_role_manifest_vars(self, ansible_dir, role, kind):
         try:
-            cluster_model = select_single(self.manifest_docs, lambda x: x.kind == 'epiphany-cluster')
-        except ExpectedSingleResultException:
+            cluster_model = self.mhandler['epiphany-cluster']
+        except IndexError:
```

Comment:
Can we have a manifest without `epiphany-cluster` kind here? If not, IMO it's better to not "hide" the exception here.